### PR TITLE
Added descriptions and meta_descriptions for cities.yml

### PR DESCRIPTION
**Pull request by AaronFried**
58 `description`
20 `meta_description`

### CODE REVIEW

File: cities.yml [hunk 1]
```diff
@@ -1,139 +1,216 @@
...
 - display_name: Austin
   slug: austin
+  meta_description: Austin coding bootcamps can help you get involved in Texas tech companies and tech jobs. Learn to code in Austin and change your career at these schools.
+  description: Learn to code in Austin, which plays an integral part in Texas' tech scene. Some coding bootcamps in Austin are among the biggest and most reputable nationwide.
 - display_name: Baltimore
   slug: baltimore
+  description: Learn to code at Baltimore coding bootcamps. Programming bootcamps in Baltimore, Maryland offer you an opportunity to code in the Midatlantic.
 - display_name: Barcelona
   slug: barcelona
+  description: Learn to code in Barcelona, of Spain's cultural centers. Coding bootcamps in Barcelona can get your programming career started in Spain.
```
Comment (tati) [inline cities.yml]: one of*

File: cities.yml [hunk 2]
```diff
@@ -1,139 +1,216 @@
...
 - display_name: Denver
   slug: denver
+  meta_description: Coding bootcamps in Denver help folks in the Mile High city get into a thriving tech industry. Learn to code in Denver and help build the future.
+  description: Learn to code in Denver, Colorado's biggest tech hub. As one of the fastest-growing tech scenes in the country, head to the Rockies, where Denver coding bootcamps include some of the biggest names in coding education.
 - display_name: Durham
   slug: durham
+  description: Learn to code in Durham, North Carolina. Conveniently neighboring Raleigh, coding bootcamps in Durham enjoy one of the Southeast's growing tech hubs.
 - display_name: Fort Collins
   slug: fort-collins
+  description: Located near major Rocky Mountain tech hubs, coding bootcamps in Fort Collins, Colorado enjoy great opportunity. Learn to code in Fort Collins, with web development and data science both available.
```
Comment (tati) [inline cities.yml]: opportunities* ?

File: cities.yml [hunk 3]
```diff
@@ -1,139 +1,216 @@
...
 - display_name: Philadelphia
   slug: philadelphia
+  description: Just blocks from the liberty bell, you can learn to code in Philadelphia. The coding bootcamps in this historic city focus on web and mobile development.
 - display_name: Phoenix
   slug: phoenix
+  description: Learn to code in Phoenix and help bring software to the desert. These Phoenix coding bootcamps focus on web development and other types of software.
 - display_name: Portland
   slug: portland
+  meta_description: Learn to code in Portland to get involved in the Northwest's growing tech industry. Portland coding bootcamps can get your software development career started.
+  description: Portland is near several tech hubs and is slowly becoming one in its own right. Learn to code in Portland coding tootcamps to get ahead on web and mobile development.
```
Comment (tati) [inline cities.yml]: tootcamps 😂

Comment (AaronFried) [inline cities.yml]: oh no 🤕

**Pull request merged**